CNV/cnv-2.1.0.sh

```diff
@@ -154,3 +154,50 @@ metadata:
   namespace: "${TARGET_NAMESPACE}"
 spec: {}
 EOF
+
+echo "Waiting for HCO to get fully deployed"
+oc wait -n ${TARGET_NAMESPACE} hyperconverged hyperconverged-cluster --for condition=Available --timeout=10m
+
+echo "Applying new network configuration on all nodes, one by one, in order to sustain quorum"
+machineCIDR=$(grep 'machineCIDR' ../OpenShift/install-config.yaml | sed 's/\(.*\): *\(.*\)/\2/')
+nodes=$(oc get nodes -o jsonpath='{range .items[*]}{.metadata.name} {end}')
+for node in $nodes; do
+    echo "Setting up node ${node} network"
+
+    while ! default_iface=$(oc get nodenetworkstate ${node} -o jsonpath="{.status.currentState.routes.running[?(@.destination==\"${machineCIDR}\")].next-hop-interface}" | cut -d " " -f 1); do
+        sleep 10
+    done
+
+    if [ "$(oc get nodenetworkstate ${node} -o jsonpath="{.status.currentState.routes.running[?(@.destination==\"${machineCIDR}\")].next-hop-interface}")" == "brext" ]; then
+        echo "Default bridged network is already set up on node ${node}"
+        break
+    fi
+
+    cat <<EOF | oc apply -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: kni-policy-${node}
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: ${node}
+  desiredState:
+    interfaces:
+    - name: brext
+      type: linux-bridge
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        dhcp: true
+        enabled: true
+      bridge:
+        port:
```

Comment:
If we don do stp enabled: false is going to enable it by default (at least this is what I have see at kubevirtci nodes).

Reply:
we have to put the stp enabled to false if not it will take time for the stp to try building the connection tree and we don't need that

Reply:
Fixed